jmix-reports/reports/src/main/java/io/jmix/reports/yarg/formatters/impl/XlsxFormatter.java

```diff
@@ -105,6 +126,38 @@ public void setFormulasPostProcessingEvaluationEnabled(boolean formulasPostProce
         this.formulasPostProcessingEvaluationEnabled = formulasPostProcessingEvaluationEnabled;
     }
 
+    private Map<String, List<XlsxImage>> loadTemplateImages() {
```

Comment:
ImagesTembletes but not TemplateImages is better as for Enlish grammar

Reply:
this methos loads images of current template so I think initial naming is correct